Build failed to fetch googleapis data from github

GCB: https://console.cloud.google.com/cloud-build/builds;region=us-east1/d82e4697-5b63-4e69-8590-66aedae4c47f;tab=detail?project=cloud-cpp-testing-resources
Raw: https://storage.googleapis.com/cloud-cpp-community-publiclogs/logs/google-cloud-cpp/main/56601817f1c8a853436c10c30360e0c75c51f94d/fedora-34-cmake-install/log-d82e4697-5b63-4e69-8590-66aedae4c47f.txt

Snippet of error:

```
T+6.333s [133/2626] Performing download step (download, verify and extract) for 'googleapis_download'
FAILED: external/googleapis/src/googleapis_download-stamp/googleapis_download-download 
cd /workspace/cmake-out/external/googleapis/src && /usr/bin/cmake -P /workspace/cmake-out/external/googleapis/src/googleapis_download-stamp/download-googleapis_download.cmake && /usr/bin/cmake -P /workspace/cmake-out/external/googleapis/src/googleapis_download-stamp/verify-googleapis_download.cmake && /usr/bin/cmake -P /workspace/cmake-out/external/googleapis/src/googleapis_download-stamp/extract-googleapis_download.cmake && /usr/bin/cmake -E touch /workspace/cmake-out/external/googleapis/src/googleapis_download-stamp/googleapis_download-download
-- Downloading...
   dst='/workspace/cmake-out/external/googleapis/src/16b43024639e5bcbb0f0367230db654f02cb8499.tar.gz'
   timeout='none'
   inactivity timeout='none'
-- Using src='https://github.com/googleapis/googleapis/archive/16b43024639e5bcbb0f0367230db654f02cb8499.tar.gz'
-- [download 100% complete]
-- [download 0% complete]
CMake Error at googleapis_download-stamp/download-googleapis_download.cmake:170 (message):
  Each download failed!

    error: downloading 'https://github.com/googleapis/googleapis/archive/16b43024639e5bcbb0f0367230db654f02cb8499.tar.gz' failed
          status_code: 22
          status_string: "HTTP response code said error"
          log:
          --- LOG BEGIN ---
            Trying 140.82.112.4:443...

  Connected to github.com (140.82.112.4) port 443 (#0)
...
```

It looks to me like a transient error w/ github servers.

--- a/bazel/google_cloud_cpp_deps.bzl
+++ b/bazel/google_cloud_cpp_deps.bzl
@@ -86,6 +86,7 @@ def google_cloud_cpp_deps():
             name = "com_google_googleapis",
             urls = [
                 "https://github.com/googleapis/googleapis/archive/343f52cd370556819da24df078308f3f709ff24b.tar.gz",
+                "https://storage.googleapis.com/cloud-cpp-community-archive/com_google_googleapis/343f52cd370556819da24df078308f3f709ff24b.tar.gz",
             ],
             strip_prefix = "googleapis-343f52cd370556819da24df078308f3f709ff24b",
             sha256 = "7248822e4d4cf4dbb4a4bc8e4eab9fc098f1be34453d7839a3f04e0a1a150ba0",

--- a/external/googleapis/CMakeLists.txt
+++ b/external/googleapis/CMakeLists.txt
@@ -18,6 +18,7 @@ include(GoogleapisConfig)
 
 set(GOOGLE_CLOUD_CPP_GOOGLEAPIS_URL
     "https://github.com/googleapis/googleapis/archive/${GOOGLE_CLOUD_CPP_GOOGLEAPIS_COMMIT_SHA}.tar.gz"
+    "https://storage.googleapis.com/cloud-cpp-community-archive/com_google_googleapis/${GOOGLE_CLOUD_CPP_GOOGLEAPIS_COMMIT_SHA}.tar.gz"
 )
 
 set(EXTERNAL_GOOGLEAPIS_SOURCE

--- a/external/googleapis/renovate.sh
+++ b/external/googleapis/renovate.sh
@@ -20,13 +20,16 @@ REPO="googleapis/googleapis"
 BRANCH="master"
 COMMIT=$(curl -sSL -H "Accept: application/vnd.github.VERSION.sha" \
   "https://api.github.com/repos/${REPO}/commits/${BRANCH}")
-SHA256=$(curl -sSL "https://github.com/${REPO}/archive/${COMMIT}.tar.gz" |
-  sha256sum | sed "s/ .*//")
+DOWNLOAD="$(mktemp)"
+curl -sSL "https://github.com/${REPO}/archive/${COMMIT}.tar.gz" -o "${DOWNLOAD}"
+gsutil -q cp "${DOWNLOAD}" "gs://cloud-cpp-community-archive/com_google_googleapis/${COMMIT}.tar.gz"
+SHA256=$(sha256sum "${DOWNLOAD}" | sed "s/ .*//")
 
 # Update the Bazel dependency.
 sed -i -f - bazel/google_cloud_cpp_deps.bzl <<EOT
   /name = "com_google_googleapis",/,/sha256 = "/ {
     s;/${REPO}/archive/.*.tar.gz",;/${REPO}/archive/${COMMIT}.tar.gz",;
+    s;/com_google_googleapis/.*.tar.gz",;/com_google_googleapis/${COMMIT}.tar.gz",;
     s/strip_prefix = "googleapis-.*",/strip_prefix = "googleapis-${COMMIT}",/
     s/sha256 = ".*",/sha256 = "${SHA256}",/
   }

--- a/external/googleapis/renovate.sh
+++ b/external/googleapis/renovate.sh
@@ -24,6 +24,7 @@ DOWNLOAD="$(mktemp)"
 curl -sSL "https://github.com/${REPO}/archive/${COMMIT}.tar.gz" -o "${DOWNLOAD}"
 gsutil -q cp "${DOWNLOAD}" "gs://cloud-cpp-community-archive/com_google_googleapis/${COMMIT}.tar.gz"
 SHA256=$(sha256sum "${DOWNLOAD}" | sed "s/ .*//")
+rm -f "${DOWNLOAD}"
 
 # Update the Bazel dependency.
 sed -i -f - bazel/google_cloud_cpp_deps.bzl <<EOT

